rubin_sim/maf/run_selfcal_metric.py

```diff
@@ -43,13 +45,22 @@ def run_selfcal_metric():
 
     # Set up the metric bundle.
     map_nside = 128
-    selfcal_metric = PhotometricSelfCalUniformityMetric(nside_residual=map_nside)
+    selfcal_metric = PhotometricSelfCalUniformityMetric(
+        nside_residual=map_nside, filter_name=args.filter, metric_name="selfcal_uniformity-%s" % args.filter
+    )
     slicer = UniSlicer()
     # Exclude DDF visits
-    sql = "note not like '%DD%'"
+    sql = "scheduler_note not like '%DD%'"
     # And run on only year 1 (?)
     sql += " and night < 366"
-    bundle = MetricBundle(selfcal_metric, slicer, sql, run_name=sim_name, info_label="year 1 no-DD")
+    sql += " and filter='%s'" % args.filter
+    # Exclude strange exposure times
+    sql += " and visitExposureTime > 20"
+    sql += " and visitExposureTime < 40"
```

Comment:
Do we actually need to drop these? It's true it makes things neater, but I would have thought including these exposures wouldn't make the self-calibration worse, and these visits ought to be calibrated as well .. 
I'd also wonder about including the DDF visits, tbh, but I think they were excluded just for calculation speed perhaps.

Reply:
I was just being extra conservative, but they should be able to be used. That will let in the twilight observations, but I think you're right that they will be weighted properly.